orb/core/connection_types/sql/postgres/statements/select.py

```diff
@@ -57,7 +57,7 @@ def __call__(self, model, context, fields=None):
                 # expand a reference
                 if isinstance(column, orb.ReferenceColumn) and column.name() in expand:
                     sub_tree = expand.pop(column.name())
-                    sql, sub_data = EXPAND_COL(column, sub_tree)
+                    sql, sub_data = EXPAND_COL(column, sub_tree, context=context)
```

Comment:
need to pass along context through the expansion in order to have it respect custom queries in the base query call